src/main/java/com/zendesk/maxwell/MaxwellAbstractRowsEvent.java

```diff
@@ -242,4 +243,33 @@ public Object getData(String string) {
 		}
 		return list;
 	}
+
+	public List<String> getPKStrings() {
+		ArrayList<String> list = new ArrayList<>();
+
+		String pkString = "db:" + table.getDatabase().getName() + "/"
+				        + "tbl:" + table.getName() + "/";
+
+		for ( Row r : filteredRows()) {
+			list.add(pkString + getPKStringForRow(r));
+		}
+		return list;
+	}
+
+	private String getPKStringForRow(Row r) {
+		if ( table.getPKList().isEmpty() )
+			return "_uuid: " + java.util.UUID.randomUUID().toString();
+
+		ArrayList<String> pkValueList = new ArrayList<>();
+		for ( String pk : table.getPKList() ) {
+			int idx = table.findColumnIndex(pk);
+
+			Column column = r.getColumns().get(idx);
+			ColumnDef def = table.getColumnList().get(idx);
+
+
+			pkValueList.add(def.getName() + ":" + def.asJSON(column.getValue()).toString());
+		}
+		return StringUtils.join(pkValueList, "/");
```

Comment:
not quite as simple as just creating a map -- you gotta also ensure key ordering... but ok.